src/lang/java/mod.rs

```diff
@@ -0,0 +1,133 @@
+use std::collections::HashMap;
+
+use crate::parse::AST;
+use crate::prophet::*;
+
+mod class_def;
+use class_def::*;
+
+mod method_body;
+mod method_def;
+mod modifier;
+mod util;
+
+/// Topmost level of the Java parser, provides public API
+
+pub fn merge_modules(modules: Vec<ModuleComponent>) -> Vec<ModuleComponent> {
+    let mut packages: HashMap<String, ModuleComponent> = HashMap::new();
+
+    // Merge same-name modules
+    for module in modules.into_iter() {
+        let name = module.module_name.clone();
+        println!("Mod. Name: {}", name);
```

Comment:
We may want to remove some of these print's at some point, once we're sure this works

Reply:
Oops, forgot this was here. I'm pretty sure it's working, but when it wasn't I was using these extensively.